libnmstate/state.py

```diff
@@ -812,6 +813,28 @@ def _normalize_linux_bridge_port_vlan(self):
                 if not port.get(LinuxBridge.Port.VLAN_SUBTREE):
                     port[LinuxBridge.Port.VLAN_SUBTREE] = {}
 
+    def _normalize_sriov_vf_mac(self):
+        """
+        SR-IOV properties are being read from the sysfs. The MAC address is
+        always being stored in lowercase, in order to avoid verification errors
+        the MAC address should be always in lowercase.
```

Comment:
In document we said MAC address is upper case but case insensitive.

I think to unify the output format of MAC address. Please make all MAC address upper case.

Reply:
Done, thanks!